src/traveler.js

```diff
@@ -0,0 +1,57 @@
+import Trip from '../src/trip.js';
+
+class Traveler {
+  constructor(travelerData) {
+    this.id = travelerData.id
+    this.name = travelerData.name
+    this.travelerType = travelerData.travelerType
+    this.allTrips = []
+    this.pastTrips = []
+    this.futureTrips = []
+    this.pendingTrips = []
+    this.currentLocation = "Denver, Colorado"
+  }
+  sayFirstName() {
+    return this.name.split(' ')[0]
+  }
+  determineTrips(trips, currentDate, allDestinations) {
+    trips.forEach(trip => {
+      let tripDestination
+      allDestinations.forEach(destination => {
+        if(trip.destinationID === destination.id)
+        tripDestination = destination
+      })
+      if (trip.userID === this.id) {
+        this.allTrips.push(new Trip(trip, tripDestination))
+      }
+    })
+    this.sortTrips(currentDate)
+  }
+  sortTrips(currentDate) {
+    this.allTrips.forEach(trip => {
+      if (trip.determineTripStatus(currentDate) === 'pending') {
+        this.pendingTrips.push(trip)
+      } else if (trip.determineTripStatus(currentDate) === 'past') {
+        this.pastTrips.push(trip)
+      } else if (trip.determineTripStatus(currentDate) === 'upcoming') {
+        this.futureTrips.push(trip)
+      } else if (trip.determineTripStatus(currentDate) === 'present') {
+        this.currentLocation = trip.destinationData.destination
+      }
+    })
+  }
+  determineYearlyTripCost(year) {
+    let yearsTrips = []
+    this.allTrips.forEach(trip => {
+      if (+trip.departDate.split('/')[0] === year) {
+        yearsTrips.push(trip)
+      }
+    })
+    let totalCost = yearsTrips.reduce((total, trip) => {
+      total += trip.determineTripCost()
+      return total
+    }, 0)
+    return +parseFloat(totalCost).toFixed(2)
```

Comment:
is the `+` on 54 supposed to be there?  If so, what purpose does it serve?

Reply:
The unary operator turns a string into a number. I keep running into trouble with parseFloat not giving me the type of information I need, but I want to use toFixed to get the cents represented for display purposes. This might be a good spot for a rethink/refactor, but this does the trick for now.